contracts/pcv/PCVGuardAdmin.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import {IPCVGuardAdmin} from "./IPCVGuardAdmin.sol";
+import {CoreRef} from "../refs/CoreRef.sol";
+import {TribeRoles} from "../core/TribeRoles.sol";
+import {ICore} from "../core/ICore.sol";
+
+/// @title PCV Guard Admin
+/// @author Volt Protocol
+/// @notice This contract interfaces between access controls and the PCV Guard
+/// allowing for multiple roles to manage the PCV Guard role, as access controls only allow for
+/// a single admin for each role
+contract PCVGuardAdmin is IPCVGuardAdmin, CoreRef {
```

Comment:
I don't understand the need for this contract. If you use `setupRoleAdmin` and grant the role-admin role out to people, each person with the admin role of the role in question would be able to give this role out.

Reply:
*governor to be able to revoke and add, and guardian be revoke only for this role, which necessitates this contract.